install/channels.csv

```diff
@@ -19,6 +19,7 @@ GEM Systems,GSM-90,Proton Magnetometer,,1,
 GEM Systems,GSM-90F1,Proton Magnetometer,,1,
 GNS Science,EARSS/3,Datalogger,,50,datalogger_GNS-Science_Earss_22bits_50sps
 GNS Science,EARSS/3,Datalogger,,100,datalogger_GNS-Science_Earss_22bits_100sps
+Kinemetrics,BASALT,Strong Motion Recorder,,1,datalogger_Kinemetrics_Basalt_26bits_1s
```

Comment:
@ozym shall we change this to be something like
`Kinemetrics,BASALT-1Hz,Geomag datalogger,,1,datalogger_Kinemetrics_Basalt_26bits_1s` ?

Reply:
no, it's still a "Strong Motion Recorder" just capable of extra things but so far only geomag uses 1hz.

Reply:
cool, thanks for clarifying